community/values/src/main/java/org/neo4j/values/ValueTuple.java

```diff
@@ -31,6 +33,18 @@ public static ValueTuple of( Value... values )
         return new ValueTuple( values );
     }
 
+    public static ValueTuple of( Object... objects )
+    {
+        assert objects.length > 0 : "Empty ValueTuple is not allowed";
+        assert noNulls( objects );
+        Value[] values = new Value[objects.length];
+        for ( int i = 0; i < values.length; i++ )
+        {
+            values[i] = Values.of( objects[i] );
+        }
+        return new ValueTuple( values );
+    }
+
     private final Value[] values;
 
     private ValueTuple( Value[] values )
```

Comment:
yeah, that's true that we have code paths that do not support composite paths so far, totally understand; what I would think that this method is even temporary till we will actually add support for composite indexes and will go away after that. I only do not like `Only` part of name since it gives impression that there is value and something else and does not really highlight that it's only for a first, single value